### PR TITLE
feat: add helm chart

### PR DESCRIPTION
**Pull request by SweetOps**

### CODE REVIEW

File: deploy/helm/falcosidekick/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+appVersion: "2.0.0"
```
Comment (Issif) [inline deploy/helm/falcosidekick/Chart.yaml]: I will release a 2.1.0 for a such good PR. Thanks.

**Comment by Issif** [comment]
@SweetOps 

I just released 2.1.0 with a new feature, we can now add custom fields to falco events, useful to add informations that falco doesn't know (region, cluster name, environment, ...). 

Can you integrate it in your .yml, I will merge after?. 

Thanks.

**Comment by SweetOps** [comment]
has been added all parameters  related to version 0.2.2

**Comment by Issif** [comment]
awesome, will review this afternoon, thanks.

**Pull request merged**